pages/[channel]/[locale]/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { ApolloQueryResult } from "@apollo/client";
+import {
+  GetStaticPaths,
+  GetStaticPropsContext,
+  InferGetStaticPropsType,
+} from "next";
+import React, { ReactElement } from "react";
+
+import { HomepageBlock, Layout } from "@/components";
+import BaseSeo from "@/components/seo/BaseSeo";
+import apolloClient from "@/lib/graphql";
+import { homepagePaths } from "@/lib/ssr/homepage";
+import { MenuQuery, MenuQueryDocument, MenuQueryVariables } from "@/saleor/api";
+
+const Home = ({ menuData }: InferGetStaticPropsType<typeof getStaticProps>) => {
+  return (
+    <>
+      <BaseSeo />
+      <div className="py-10">
+        <header className="mb-4">
+          <div className="max-w-7xl mx-auto px-8"></div>
+        </header>
+        <main>
+          <div className="max-w-7xl mx-auto px-8">
+            {menuData?.menu?.items?.map((m) => {
+              if (!!m) return <HomepageBlock key={m?.id} menuItem={m} />;
```

Comment:
```suggestion
              if (m) return <HomepageBlock key={m?.id} menuItem={m} />;
```
value is already cast to boolean inside condition

Reply:
Used pattern existing in the rest of the project. Leaving it to be resolved in separate PR.